### ISSUE TITLE
Unable to style tooltip in CommandBar

### ISSUE THREAD

**Issue by davallen** [microsoft/fluentui#16147]
Issue: 

I am having contrast issue with tooltip that is wrapping the button inside commandbar. My command bar is a blue color, but the highlighted dive in the screenshot is getting flagged in accessibility for contrast issues as it see the background of the command bar and the text color. I have tried the root, content and subText section of ToolTipStyles to try and manipulate this div, but nothing is working.

![image](https://user-images.githubusercontent.com/19310619/101209441-97e6a900-3628-11eb-8781-0e6c8bdfcd7e.png)

#### Actual behavior:

Unable to set the background color of the div with id="tooltip<number>" and it is getting flagged in accessibility check

#### Expected behavior:

Should be able to change the background color of the div

**Comment by JustSlone** [comment]
@davallen thanks for reaching out! Can you share a CodePen/CodeSandbox showing the code you have, there might be a way to accomplish the styling you want. Without seeing the code and repro using Fluent UI React controls, it's hard to figure out what will fix it though.
